legal-api/src/legal_api/models/business.py

```diff
@@ -267,6 +271,42 @@ def identifier(self, value: str):
         else:
             raise BusinessException('invalid-identifier-format', 406)
 
+    @hybrid_property
+    def business_legal_name(self):
+        """
+        Return correct legal name based on feature flag.
+
+        Legal Name Easy Fix
+        """
+        from legal_api.services import flags  # pylint: disable=import-outside-toplevel
+
+        flag_on = flags.is_on('enable-legal-name-fix')
+        if self.is_firm and flag_on:
+            sort_name = func.trim(
+                func.coalesce(Party.organization_name, '') +
+                func.coalesce(Party.last_name+' ', '') +
+                func.coalesce(Party.first_name+' ', '') +
+                func.coalesce(Party.middle_initial, '')
+            )
+
+            parties_query = self.party_roles.join(Party).filter(
+                func.lower(PartyRole.role).in_([
+                    func.lower(value) for value in [
+                        PartyRole.RoleTypes.PARTNER.value,
+                        PartyRole.RoleTypes.PROPRIETOR.value
```

Comment:
Just curious why `func.lower` is needed here. Aren't these enum values already lower case?

Reply:
Sure, let me check 😸